core/sync.js

```diff
@@ -297,6 +297,9 @@ class Sync {
       } else {
         await this.doMove(side, doc, from)
       }
+      if (!sameBinary(from, doc)) {
+        await side.overwriteFileAsync(doc, from)
+      }
```

Comment:
It doesn't seem to fix the scenario on CI:
https://travis-ci.org/cozy-labs/cozy-desktop/jobs/454611472#L377

I tried every combo of `isUpToDate` / `isAtLeastUpToDate`, but none of them fix it either.
So I assume something else in #1250 was making it work.

Reply:
Looks like it, let's merge #1274 and then we will see.

Reply:
I opened a PR with only this fix to see if it's sufficient in https://github.com/cozy-labs/cozy-desktop/pull/1290/files